### PR TITLE
Add all items counter on the Homepage

### LINKED ISSUE #6: [3pt]  - Add all items counter on the Homepage - Student A
Even if the API gives you the number of items, you will create a specific function to calculate it.

Make sure that the correct number is displayed on the Homepage.

Respect the following rules:

- Each counter should be implemented as a separate module.
- A counter function should look for specific DOM elements (e.g. for the comments counter it should look for comments) and make the counting based on what is actually displayed on the page.
- A counter function should cover all the edge cases you can think about.

### PR DESCRIPTION
**Pull request by AndreaM2429**
# _**Add all items counter on the Homepage**_

Hello @ClaudiaRojasSoto, I ask for your approve to merge this changes into the `develop` branch. This merge is about the function to display the correct number of items in the home page.

## _**Milestones**_ 🎯

The milestones this time are:

- [ ] Create a new function called `homeItemCounter()` that is in the `src/Modules/itemCounter.js` path. This function display the number of items in the home page.
- [ ] Display the correct number in the `nav` next to `Home` link.
- [ ] In the `index.html` I added a ID for the DOM Manipulation.

I appreciate any comment or suggestion to improve the code and functionality. I will waiting for your review, thanks in advance.

### CODE REVIEW

**Review by ClaudiaRojasSoto — CHANGES_REQUESTED** [review]
@AndreaM2429 Your work looks great, congratulations:

### Highlights:
- [x] Your feature it's working as expected.
- [x] You followed these best practices on JavaScript.
- [x] You followed the design and the layout given by Microverse.
 
However, for the work to gain user experience, I have a suggestion that you can read above.

congrats

File: src/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <header class="dflex">
         <img id="logo" alt="Vegetarian Food">
         <ul class="dflex">
-            <li><a href="#">Home</a></li>
+            <li><a id="home" href="#">Home</a></li>
```
Comment (ClaudiaRojasSoto) [inline src/index.html]: - [ ] Could you change the ubication of the item counter from nav to body?

> This change will substantially improve the user experience.

Comment (AndreaM2429) [reply src/index.html]: Hello @ClaudiaRojasSoto, thanks for taking the time to review my code.
I will implement the changes you ask.

**Review by ClaudiaRojasSoto — APPROVED** [review]
Your work´s complete. 🥇 
Due to your hard work, you can merge it.
Congratulations!!!!! 💯 

congrats

**Pull request merged**